### DISCUSSION TITLE
Some FFMPEG -vf (src) Filter codes to share...

### DISCUSSION THREAD

**Discussion by diverswan9** [nagadomi/nunif#258] [Ideas]
Here' some FFMPEG -vf (src) Filter codes that I put on a text file. It is by no means a 'Comprehensive' list... just sort of a 'work in progress'.

`https://drive.google.com/file/d/1W8DtbBSioowrr-ETG6k-g5p0rzum5wvc/view?usp=drive_link`

**Comment by andy500** [comment]
can not get this to work  Some CMD window stuff mostly for iw3 'Export Disparity" (open CMD in target folder)

cmd for converting .png to .mp4
        ffmpeg -r 23.975 -i %08d.png. -c:v libx264 -crf 15 out.mp4 how to get it to work

**Reply by diverswan9** [reply]
It should work... FFMPEG needs to be installed in your 'environment' and it's for 'Windows'. Run 'CMD' as Admin from target folder.

 It will create an mp4 file in the same folder named "out.mp4".

There's also another version at the bottom of that text page.

**Reply by andy500** [reply]
this is the error l am having 
  built with gcc 14.2.0 (Rev1, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-bzlib --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libaribb24 --enable-libaribcaption --enable-libdav1d --enable-libdavs2 --enable-libopenjpeg --enable-libquirc --enable-libuavs3d --enable-libxevd --enable-libzvbi --enable-libqrencode --enable-librav1e --enable-libsvtav1 --enable-libvvenc --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxavs2 --enable-libxeve --enable-libxvid --enable-libaom --enable-libjxl --enable-libvpx --enable-mediafoundation --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libharfbuzz --enable-liblensfun --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-dxva2 --enable-d3d11va --enable-d3d12va --enable-ffnvcodec --enable-libvpl --enable-nvdec --enable-nvenc --enable-vaapi --enable-libshaderc --enable-vulkan --enable-libplacebo --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libcodec2 --enable-libilbc --enable-libgsm --enable-liblc3 --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      59. 39.100 / 59. 39.100
  libavcodec     61. 19.100 / 61. 19.100
  libavformat    61.  7.100 / 61.  7.100
  libavdevice    61.  3.100 / 61.  3.100
  libavfilter    10.  4.100 / 10.  4.100
  libswscale      8.  3.100 /  8.  3.100
  libswresample   5.  3.100 /  5.  3.100
  libpostproc    58.  3.100 / 58.  3.100
[image2 @ 000002190e0dab00] Could find no file with path '%08d.png' and index in the range 0-4
[in#0 @ 000002190e0da700] Error opening input: No such file or directory
Error opening input file %08d.png.
Error opening input files: No such file or directory

**Reply by diverswan9** [reply]
Are you using Windows OS?

Are using latest FFmpeg?

Lauching CMD as admin from within the png folder?

I assume the png files are either in the 'depth' folder or the 'rgb' folder and have '8' digits? (i.e. - 00000000.png)

Try running the other code at the bottom of the text?
`ffmpeg -framerate 24000/1001 -i %08d.png -vf scale=1920:800 -pix_fmt yuv420p -c:v h264_nvenc -b:v 15000k depthmap.mp4`
